src/main/java/frc/robot/Constants.java

```diff
@@ -224,50 +233,69 @@ public static final class CoralManipulator {
+        public static final double INTAKE_SPEED = 0.2;
+        public static final double SCORE_SPEED = 0.2;
         public static final double HOLD_SPEED = 0;
     }
   
     public static class AlgaeIntake {
         public static final Dimensionless INTAKE_GEAR_RATIO = Rotations.of(15).div(Rotations.of(1)); // output over input
-        public static final double INTAKE_SPEED = 0.9;
+        public static final double INTAKE_SPEED = 0.2;
 
-        public static final Angle MAX_ERROR = Degrees.of(5.0);
-        public static final Angle RAISED_POS = Degrees.of(199.5).div(INTAKE_GEAR_RATIO);
-        public static final Angle LOWERED_POS = Degrees.of(0);
+        public static final Angle MAX_ERROR = Degrees.of(5.0).times(INTAKE_GEAR_RATIO);
+        public static final Angle RAISED_POS = Degrees.of(12).times(INTAKE_GEAR_RATIO);
+        public static final Angle LOWERED_POS = Degrees.of(53).times(INTAKE_GEAR_RATIO);
 
-        public static final Current CURRENT_LIMIT = Amps.of(35.0);
+        public static final Current CURRENT_SPIKE_LIMIT_DOWN = Amps.of(5);
+        public static final Current CURRENT_SPIKE_LIMIT_UP = Amps.of(5);
 
         //TODO: use sysid and set all of these values
         public static final MotorConfig INTAKE_CONFIG = new MotorConfig(
-            Ports.INTAKE_ROLLER_ID,
-            0,
+            Ports.ALGAE_INTAKE_ROLLER_ID,
+            35,
             true, 
             MotorConfig.Mode.COAST
         );
 
         public static final MotorConfig PIVOT_CONFIG = new MotorConfig(
             Ports.INTAKE_PIVOT_ID,
-            0,
+            35,
             true,
-            PIDConfig.getArmPid(0.0, 0.0, 0.0, 0, 0, 0, 0),
+            PIDConfig.getArmPid(0.2, 0.0, 0.0, 0, 0, 0, 0),
             MotorConfig.Mode.BRAKE
         );
     }
     
-    public static final class AutoConstants { //TODO: Need to tune constants
+    public static final class AutoConstants { //TODO: Need to tune constants!
         public static final double kMaxSpeedMetersPerSecond = 3;
         public static final double kMaxAccelerationMetersPerSecondSquared = 3;
         public static final double kMaxAngularSpeedRadiansPerSecond = Math.PI;
         public static final double kMaxAngularSpeedRadiansPerSecondSquared = Math.PI;
     
-        public static final double kPTranslationController = 1;
-        public static final double kPThetaController = 1;
+        public static final double kPTranslationController = .2;
+        public static final double kPThetaController = .2;
+    }
+
+    public static final class VisionConstants { //TODO: only tell pipeline to give pose when multiple tags detected
+        public static final Transform3d leftTransform3d = new Transform3d(new Translation3d(7.6724, 11.7981, 7.291), new Rotation3d(0, 20, -36.5));
+        public static final Transform3d rightTransform3d = new Transform3d(new Translation3d(7.6724, -11.7981, 7.291), new Rotation3d(0, 20, 36.5));
```

Comment:
```suggestion
        public static final Transform3d leftCamera = new Transform3d(new Translation3d(7.6724, 11.7981, 7.291), new Rotation3d(0, 20, -36.5));
        public static final Transform3d rightCamera = new Transform3d(new Translation3d(7.6724, -11.7981, 7.291), new Rotation3d(0, 20, 36.5));
```
in general, don't include type information in a variable's name